Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -3710,11 +3710,11 @@ def gen2021HiMix(fragment,howMuch):
 defaultDataSets['2017Design']='CMSSW_12_0_0_pre4-113X_mc2017_design_v5-v'
 defaultDataSets['2018']='CMSSW_12_0_0_pre4-113X_upgrade2018_realistic_v5-v'
 defaultDataSets['2018Design']='CMSSW_12_0_0_pre4-113X_upgrade2018_design_v5-v'
-defaultDataSets['2021']='CMSSW_12_0_0_pre4-120X_mcRun3_2021_realistic_v2-v'
-defaultDataSets['2021Design']='CMSSW_12_0_0_pre4-120X_mcRun3_2021_design_v2-v'
-defaultDataSets['2021FS']='CMSSW_12_4_0_pre4-124X_mcRun3_2021_realistic_v1_Run3FastSim_FastSim-v'
-defaultDataSets['2023']='CMSSW_12_0_0_pre4-120X_mcRun3_2023_realistic_v2-v'
-defaultDataSets['2024']='CMSSW_12_0_0_pre4-120X_mcRun3_2024_realistic_v2-v'
+defaultDataSets['2021']='CMSSW_12_5_0_pre4-124X_mcRun3_2022_realistic_v10_realistic-v1'
```

Comment:
I think we should use `-v` not `-v1`
This applies to all lines.

Reply:
Now reproducing these samples with also dropping the custom label, which resulted in double naming in the output datasets.

Reply:
Hi @kskovpen 

For FullSim, If you want to avoid complication later, I think keeping `CMSSW_12_5_0_pre4-124X_mcRun3_2022_realistic_v10_realistic-v` is maybe the easiest way. 

If not, maybe inject with custom label that make more clear, e.g. 2022BS. So that, it will reflect that this MinBias dataset is not produced from default config of CMSSW_12_5_0_pre4.

But if prefer v2, version overwrite maybe an option to go.

Reply:
Hi @srimanob , thanks for your suggestions. I think I ran into some problem when dropping the version number, but let's try again once the new FS MinBias is ready (should be done by tomorrow).